src/utils.jl

```diff
@@ -35,6 +31,7 @@ rather than `LowerTriangular`.
     that returns a `Matrix` rather than `LowerTriangular`.
 """
 cholesky_lower(X::AbstractMatrix) = lower_triangular(parent(cholesky(Hermitian(X, :L)).L))
+cholesky_lower(X::PDMat) = X.chol.L
```

Comment:
To avoid relying on internals of PDMats:
```suggestion
cholesky_lower(X::PDMat) = cholesky_lower(cholesky(X))
```